ui/lib/core/addon/components/form-field.js

```diff
@@ -75,6 +75,11 @@ export default class FormFieldComponent extends Component {
     this.showInput = !!modelValue;
   }
 
+  get hasRadioSubTextAndLabel() {
+    // for 'radio' editType, check to see if every of the possibleValues has a subText and label
+    return this.args?.attr?.options?.possibleValues?.every((v) => v.subText && v.label);
```

Comment:
why do we need to check for label? the styling changes that are made only seem necessary if `subtext` is passed?

Reply:
Looking at HDS radio in Helios, there isn't an example of mixing the two. However, I do notice that we can pass label and an optional helper.

Reply:
I don't follow 😅 

I think this conditional should just be checking for subtext for one option. I'm wouldn't expect HDS to have an example of that. The way the conditional is written right now, an option could be using the `value` key as a label and have subtext that doesn't render 
`this.args?.attr?.options?.possibleValues?.any((v) => v.subText );`

Reply:
I updated it. :) I was looking at the code examples and didn't see an example but the [component](https://hds-website-hashicorp.vercel.app/components/form/radio?tab=code#field-items) will allow for an optional helper text.

<img width="308" alt="Screenshot 2023-12-14 at 10 06 22 AM" src="https://github.com/hashicorp/vault/assets/30884335/f83e3ce2-2114-4184-9987-328a0a885c23">

Reply:
Yep - which is the functionality we're adding here, right? Passing in optional subtext for each option